docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -222,62 +224,113 @@ endif::[]
-. Click *Import*. The design editor opens automatically.
+.. *Import from {registry}*: From the instances list, select a {registry} instance.
+Browse the list of artifacts for that instance, and select an artifact.
+. When prompted, provide additional information such as version (_OpenAPI only_), name, type, and summary in the import dialog. You can overwrite any autopopulated values.
+. Click *Import*.
+The design editor opens automatically.
 
-When you finish editing, you can export the updated design to {registry} as a new artifact or as a new version of the existing artifact. You can also save your changes locally, or download the content to a file.
+When you finish editing, you can export the updated design to {registry} as a new artifact or as a new version of the existing artifact.
+You can export the design from the design editor or from the *API and Schema Designs* page.
+You can also save your changes locally, or download the content to a file.
 
 .Verification
 ifdef::qs[]
-* Is the imported design listed on the {product-api-designer} designs page?
+* Is the imported design listed on the designs page?
 * Are the design details correct?
 endif::[]
 ifndef::qs[]
-. Verify that the imported design is listed on the {product-api-designer} designs page.
+. Verify that the imported design is listed on the designs page.
 . Verify that the design details are correct.
 endif::[]
 
+[id="proc-testing-schema-api-design_{context}"]
+== Testing a schema or API design
+
+[role="_abstract"]
+You can test that your schema or API design content is valid and compatible with existing content by applying the rules of an existing {product-long-registry} instance.
+You can test the design while working in the {product-long-api-designer} editor, without exporting the design to {product-long-registry}.
+
+.Prerequisites
+* You're logged in to the {product-long-api-designer} web console at {service-url-api-designer}[^].
+* You've created or imported an {product-api-designer} design.
+* You can access a running {product-registry} instance in your organization.
+Within that instance, you know the artifact ID of an artifact with the required rules configured.
+For more information, see {base-url}{getting-started-url-registry}[Getting started with {product-long-registry}^].
+
+.Procedure
+. In the *API and Schema Designs* page of the {service-url-api-designer}[{product-long-api-designer} web console^], select the design that you want to test.
+. Click *Edit* to open the design editor.
+. From the *Actions* menu, click *Test using Service Registry*.
+. From the *Registry instance* list, select a {registry} instance.
+. Specify the artifact details in the *Group* and *Artifact ID* fields.
+. Click *Test*.
++
+NOTE: Note that the UI text is in flux and might change.
+
+The test results are shown in a new pane in the design editor window:
```

Comment:
Can probably rewrite this to eliminate passive voice:

"A new pane in the design editor window shows the test results."

Reply:
Done